yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -553,6 +552,35 @@ func (p *ParserIssueDetector) FinalizeColumnMetadata() {
 
 	// Build partition hierarchies
 	p.buildPartitionHierarchies()
+
+	p.buildUsageCategoryForAllTables()
```

Comment:
`FinalizeColumnMetadata` method name seems a bit confusing. From what I can see, it seems to be run after all DDLs in TABLE file are processed. 

Should we call it something like `FinalizeTablesMetadata` since a lot of other tthings like fk constraints/partition hierarchies/usage category(now) are being computed/populated here).